### DISCUSSION TITLE
copy/paste和系统剪贴板是不通的？

### DISCUSSION THREAD

**Discussion by RuralHunter** [kingToolbox/WindTerm#1186] [Q&A]
2.6.0，linux版ubuntu，所有copy/paste只在windterm里自己用，和系统剪贴板不通，即windterm里copy的内容无法粘贴到其它应用，怎么才能和系统剪贴板连通？

**Comment by RuralHunter** [comment]
我发现了，是设置了鼠标右键直接粘贴功能影响的。如果鼠标右键设置是弹出菜单，那么通过菜单复制到其它应用是没问题的。
所以在设置鼠标右键是粘贴的情况下，可能是鼠标左键选择时实际上并没有复制，只有在按右键的时候才复制？
可是也不对，鼠标右键是可以直接粘贴从其它应用复制到剪贴板的东西的。总之这里边有矛盾。

**Comment by RuralHunter** [comment]
我有点明白了，鼠标右键直接粘贴时根本没有用到剪贴板，所以我需要的是一个改动或者新功能：选择即复制的功能，我用过的其它term软件都是这么实现的。

**Reply by hale177** [reply]
测了 ，我也是这样，外面应用可以粘贴到 windterm ，反之不行

**Comment by JFPeak** [comment]
Windterm2.7.0
把Ctrl+Shift+C和Ctrl+Shift+V分别配置为复制和粘贴。方法：
在windterm安装路径下，编辑修改 global/windterm.keymaps 文件里面的这两部分，重启windterm即可生效

{ "keys": "<Ctrl+Shift+C>",
    "modes": "normal, command, local, remote",
    "action": "Text.Copy"
  },
  { "keys": "<Ctrl+Shift+V>",
    "modes": "normal, command, local, remote",
    "action": "Text.Paste"
  },
